attributes/wsus_client.rb

```diff
@@ -20,7 +20,7 @@
 return unless platform?('windows')
 
 # Defines whether a custom WSUS server should be used instead of Microsoft Windows Update server.
-default['wsus_client']['wsus_server']                              = 'http://kitchen-051e26df5045156e.centralus.cloudapp.azure.com:8530'
+default['wsus_client']['wsus_server']                              = 'http://wsus.example.com:8530'
```

Comment:
Is this going to spit out an error message in the system log until it is adjusted to a valid update server? Maybe this is valid because it makes sure we can walk a user through proper configuration?

Reply:
Also left wondering if it is best security practice to set this to a "valid" domain like example.com or if we should use something more 'broken' by default like CHANGEME?

Reply:
It won't throw much in the way of errors as that part of the wsus-client cookbook writes that string to a registry key. This was more of a "here's the format you should use" kind of example. This is something we should call out in the setup.md docs